test/data/pdep_err_test.yaml

```diff
@@ -0,0 +1,263 @@
+description: |-
```

Comment:
Can you use the absolute minimum file that produces the condition you want to test, including removing all the comments that aren't related to the test?